ibis/backends/tests/test_temporal.py

```diff
@@ -317,6 +331,16 @@ def test_timestamp_extract_milliseconds(backend, alltypes, df):
+    ["flink"],
+    raises=AssertionError,
+    reason=(
+        "E   Series values are different (100.0 %)"
+        "E   [index]: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, ...]"
+        "E   [left]:  [1288584000, 1288584060, 1288584120, 1288584180, 1288584240, 1288584300, 1288584360, 1288584420, 1288584480, 1288584540, 1288671000, 1288671060, 1288671120, 1288671180, 1288671240, 1288671300, 1288671360, 1288671420, 1288671480, 1288671540, 1288758000, 1288758060, 1288758120, 1288758180, 1288758240, 1288758301, 1288758361, 1288758421, 1288758481, 1288758541, 1288845001, 1288845061, 1288845121, 1288845181, 1288845241, 1288845301, 1288845361, 1288845421, 1288845481, 1288845541, 1288932001, 1288932061, 1288932121, 1288932181, 1288932241, 1288932301, 1288932361, 1288932422, 1288932482, 1288932542, 1289019002, 1289019062, 1289019122, 1289019182, 1289019242, 1289019302, 1289019362, 1289019422, 1289019482, 1289019542, 1289109602, 1289109662, 1289109722, 1289109782, 1289109842, 1289109902, 1289109962, 1289110022, 1289110082, 1289110143, 1289196603, 1289196663, 1289196723, 1289196783, 1289196843, 1289196903, 1289196963, 1289197023, 1289197083, 1289197143, 1289283603, 1289283663, 1289283723, 1289283783, 1289283843, 1289283903, 1289283963, 1289284023, 1289284083, 1289284143, 1289370604, 1289370664, 1289370724, 1289370784, 1289370844, 1289370904, 1289370964, 1289371024, 1289371084, 1289371144, ...]"
+        "E   [right]: [1288569600, 1288569660, 1288569720, 1288569780, 1288569840, 1288569900, 1288569960, 1288570020, 1288570080, 1288570140, 1288656600, 1288656660, 1288656720, 1288656780, 1288656840, 1288656900, 1288656960, 1288657020, 1288657080, 1288657140, 1288743600, 1288743660, 1288743720, 1288743780, 1288743840, 1288743901, 1288743961, 1288744021, 1288744081, 1288744141, 1288830601, 1288830661, 1288830721, 1288830781, 1288830841, 1288830901, 1288830961, 1288831021, 1288831081, 1288831141, 1288917601, 1288917661, 1288917721, 1288917781, 1288917841, 1288917901, 1288917961, 1288918022, 1288918082, 1288918142, 1289004602, 1289004662, 1289004722, 1289004782, 1289004842, 1289004902, 1289004962, 1289005022, 1289005082, 1289005142, 1289091602, 1289091662, 1289091722, 1289091782, 1289091842, 1289091902, 1289091962, 1289092022, 1289092082, 1289092143, 1289178603, 1289178663, 1289178723, 1289178783, 1289178843, 1289178903, 1289178963, 1289179023, 1289179083, 1289179143, 1289265603, 1289265663, 1289265723, 1289265783, 1289265843, 1289265903, 1289265963, 1289266023, 1289266083, 1289266143, 1289352604, 1289352664, 1289352724, 1289352784, 1289352844, 1289352904, 1289352964, 1289353024, 1289353084, 1289353144, ...]"
```

Comment:
The messages in reason shouldn't contain this entire pytest diff, you can include the first line and that's enough.

Reply:
Done; following up to understand why it's broken in https://github.com/ibis-project/ibis/pull/6920#discussion_r1375464423